compass/landice/tests/ismip6_forcing/atmosphere/process_smb.py

```diff
@@ -0,0 +1,331 @@
+        output_file = f"processed_SMB_{model}_{scenario}_{period_endyear}.nc"
+        self.rename_ismip6smb_to_mali_vars(remapped_file_temp, output_file)
+
+        # correct the SMB anomaly field with mali base SMB field
+        logger.info("Correcting the SMB anomaly field with MALI base SMB "
+                    "from file {mali_mesh_file}...")
+        self.correct_SMB_anomaly_for_baseSMB(output_file, mali_mesh_file)
+
+        logger.info("Processing done successfully. "
+                    "Removing the temporary files 'remapped.nc' and "
+                    "'combined.nc'...")
+        # remove the temporary remapped and combined files
+        os.remove(remapped_file_temp)
+        os.remove(combined_file_temp)
+
+        # place the output file in appropriate directory
+        if output_base_path == "NotAvailable":
+            return
+
+        output_path = f'{output_base_path}/atmosphere_forcing/' \
+                      f'{model}_{scenario}/1995-{period_endyear}'
+        if not os.path.exists(output_path):
+            print("Creating a new directory for the output data")
+            os.makedirs(output_path)
+
+        src = os.path.join(os.getcwd(), output_file)
+        dst = os.path.join(output_path, output_file)
+        shutil.copy(src, dst)
+
+    def remap_ismip6smb_to_mali(self, input_file, output_file, mali_mesh_name,
+                                mali_mesh_file, method_remap):
+        """
+        Remap the input ismip6 thermal forcing data onto mali mesh
+
+        Parameters
+        ----------
+        input_file: str
+            ismip6 smb data on its native polarstereo 8km grid
+        output_file : str
+            ismip6 data remapped on mali mesh
+        mali_mesh_name : str
+            name of the mali mesh used to name mapping files
+        mali_mesh_file : str, optional
+            The MALI mesh file if mapping file does not exist
+        method_remap : str, optional
+            Remapping method used in building a mapping file
+        """
+
+        # check if a mapfile exists
+        mapping_file = f"map_ismip6_8km_to_{mali_mesh_name}_{method_remap}.nc"
+
+        if not os.path.exists(mapping_file):
+            # build a mapping file if it doesn't already exist
+            self.logger.info("Creating a mapping file. "
+                             "Mapping method used: {method_remap}")
+            build_mapping_file(self.config, self.cores, self.logger,
+                               input_file, mapping_file, mali_mesh_file,
+                               method_remap)
+        else:
+            self.logger.info("Mapping file exists. "
+                             "Remapping the input data...")
+
+        # remap the input data
+        args = ["ncremap",
+                "-i", input_file,
+                "-o", output_file,
+                "-m", mapping_file,
+                "-v", "smb_anomaly"]
+
+        subprocess.check_call(args)
```

Comment:
Again, this needs to use `mpas_tools.logging.check_call` and pass `self.logger`.  Otherwise, output goes to the terminal instead of the log file.